docs/source-verification.md

```diff
@@ -0,0 +1,177 @@
+
+# Instructions for the public
+
+_Don't trust, verify!_
+
+## Verify through Blockscout
+
+The default page for any address link in Blockscout is the "Transactions" tab. If the address is a a smart contract, you can click to the "Code" tab. Here's an example of an _unverified_ smart contract with simply the deployed bytecode visible. There is a button to "Verify & Publish". Here's an example from: https://alfajores-blockscout.celo-testnet.org/address/0x627d7afdd37c94e0badc0ce48bf9d3d857fd9b2c/contracts
+
+![Unverified Smart Contract](./unverified.png)
+
+If the project has already gone through the verification steps, in addition to the deployed bytecode, the source code, compiler version, EVM version and links to the Sourcify repository will appear. The "Verify & Publish" button is now hidden. Here's an example from: https://alfajores-blockscout.celo-testnet.org/address/0x4485b3db2e2b5b43aafa104a01cbee829573e806/contracts
+
+![Verified Smart Contract](./verified.png)
+
+## Verify through Sourcify.dev
+
+You may also skip the Blockscout page and go directly to Sourcify (https://sourcify.dev) and check for yourself. Select the network from the dropdown list, enter the address of the contract you want to verify
+
+![Verified Smart Contract Sourcify](./verified_sourcify.png)
+
+Follow the link in the result header, or just navigate to the Sourcify repository: https://repo.sourcify.dev/ and browse for the chainID under the `contracts/full_match` folder. In this example, the chainID is `44787` for the Celo Alfajores testnet. Example: https://repo.sourcify.dev/contracts/full_match/44787/0x4485B3db2E2B5b43aAfA104a01cBEe829573e806/sources/home/boyd/git/keyko/celo-ubi-contract/contracts/
+
+![Sourcify Code](./sourcecode_sourcify.png)
+
+## Does a verified contract mean it is safe to interact with?
+
+## _No!_
+Verification is simply a way of proving that a smart contract developer has deployed bytecode to a certain on-chain address that matches a _published_ and provably known source. Source code is stored on IPFS which can be retrieved by anyone manually even if Sourcify ceased to exist tomorrow.
+
+Verification does not make any claims to the _function_, security, or fitness for any particular purpose of any smart contract. It is simply one piece of the decentralized security landscape. The public should inspect verified source code and check that it does what the project claims it should.
+
+# Instructions for smart contract deployers
+
+## General Requirements
+
+* Verification requires the generated JSON artifact created during the build process (e.g. `/build/Contract.json`) _and_ the source Solidity file (e.g. `./contracts/Contract.sol`)
+* However, if you set the `solc` metadata option `useLiteralSources`, you can skip providing the Solidity file and simply upload the JSON artifact, as the source code is already embedded in the build output file. 
+
+A sample build configuration snippet for Celo (from `truffle-config.json`):
```

Comment:
@yorhodes mentioned that the file is actually called `truffle-config.js`, see [here](https://www.trufflesuite.com/docs/truffle/reference/configuration).

Reply:
Fixed